modin/engines/ray/pandas_on_ray/frame/partition_manager.py

```diff
@@ -204,13 +288,64 @@ def map_axis_partitions(
         lengths=None,
         enumerate_partitions=False,
     ):
+        """
+        Apply `map_func` to every partition in `partitions` along given `axis`.
+
+        Parameters
+        ----------
+        axis : {0, 1}
+            Axis to perform the map across (0 - index, 1 - columns).
+        partitions : np.ndarray
+            A NumPy 2D array of partitions to perform operation on.
+        map_func : callable
+            Function to apply.
+        keep_partitioning : bool, default: False
+            Whether to keep partitioning for Modin Frame.
+            Setting it to True stops data shuffling between partitions.
+        lengths : list of ints, default: None
+            List of lengths to shuffle the object.
+        enumerate_partitions : bool, default: False
+            Whether or not to pass partition index into `map_func`.
+            Note that `map_func` must be able to accept `partition_idx` kwarg.
+
+        Returns
+        -------
+        np.ndarray
+            A NumPy array of new partitions for Modin Frame.
+
+        Notes
+        -----
+        This method should be used in the case when `map_func` relies on
+        some global information about the axis.
+        """
         return super(PandasOnRayFrameManager, cls).map_axis_partitions(
             axis, partitions, map_func, keep_partitioning, lengths, enumerate_partitions
         )
 
     @classmethod
     @progress_bar_wrapper
     def _apply_func_to_list_of_partitions(cls, func, partitions, **kwargs):
+        """
+        Apply a `func` to a list of remote `partitions`.
+
+        Parameters
+        ----------
+        func : callable
+            The func to apply.
+        partitions : np.ndarray
+            The partitions to which the `func` will apply.
+        **kwargs : dict
+            Keyword arguments.
+
+        Returns
+        -------
+        list
+            A list of ``RayFramePartition`` objects.
+
+        Notes
+        -----
+        The main use for this is to preprocess the func.
```

Comment:
I wonder why `cls.preprocess_func()` is not enough in this case...

Reply:
Probably, part of methods of this class makes simple calls of parent-class methods, because methods in this class is wrapped by progress bar.

Reply:
It seems this is not correct note for the method. Right note would be that ([link](https://github.com/modin-project/modin/blob/master/modin/engines/base/frame/partition_manager.py#L920)). We should fix it.

Reply:
@YarShev fixed